web/src/package.json

```diff
@@ -66,7 +66,7 @@
     "@apollo/react-hooks": "3.1.5",
     "@babel/runtime": "7.9.6",
     "@date-io/luxon": "1.3.13",
-    "@material-ui/core": "4.9.8",
+    "@material-ui/core": "4.10.2",
```

Comment:
It looks like there is a dependabot PR to update to 4.11 already -- if this PR needs this update, let's get the MUI PR in first so this one doesn't need to do any of it's own package updates.

If it doesn't then let's update `package.json` and `yarn.lock` to use the versions from master rather than introducing the update on this PR.

That way it will be easier to identify if any test regressions are from the UI update or the changes specific to this PR.

Reply:
Fixed with PR #786